client/src/components/BackgroundImages.ts

```diff
@@ -0,0 +1,27 @@
+import { IBackgrounds } from '../types';
+import butterflys from '../assets/backgrounds/butterflys.jpg';
+import grey_board from '../assets/backgrounds/grey_board.jpg';
+import ice from '../assets/backgrounds/ice.jpg';
+import mountain from '../assets/backgrounds/mountain.jpg';
+import nature from '../assets/backgrounds/nature.jpg';
+import night_sky from '../assets/backgrounds/night_sky.jpg';
+import stones from '../assets/backgrounds/stones.jpg';
+import tiles from '../assets/backgrounds/tiles.jpg';
+import tree from '../assets/backgrounds/tree.jpg';
+import triangles from '../assets/backgrounds/triangles.png';
+import blank from '../assets/backgrounds/blank.png';
```

Comment:
importing a white image is a bad idea
- we can do this in css which is much better than loading an image, and css will be higher quality than an image since it can scale infinitely
- if we want to change the color scheme for example white on black instead of black on white, we'd have to get a new image which isn't a big deal but much easier to change a css value

Instead, you should handle the case of 'blank' by setting the background as white or undefined